workers/ohsome_quality_analyst/oqt.py

```diff
@@ -107,6 +112,9 @@ async def create_report_as_geojson(
             logging.info("Input feature identifier: " + id_)
             if size_restriction:
                 await check_area_size(feature.geometry)
+            # Reports for a FeaturesCollection are not created asynchronously (as it is
+            # the case with indicators), because indicators of a report are created
+            # asynchronously
```

Comment:
```suggestion
            # Reports for a FeatureCollection are not created asynchronously (as it is
            # the case with indicators), because indicators of a report are created
            # asynchronously
```